src/http/responses/asset-info.response.ts

```diff
@@ -21,6 +21,10 @@ export class AssetInfo {
     @ApiProperty()
     @Expose()
     apy?: number;
+
+    @ApiProperty()
+    @Expose({ name: 'totalAllocatedToken' })
```

Comment:
Why is this required?

Reply:
thought of changing the alias name for the exposed property. But true we don't do it for the others in the class. I remove it.